tools/clang/unittests/HLSL/CompilerTest.cpp

```diff
@@ -1362,9 +1361,9 @@ static void VerifyPdbUtil(
 
   // Main file name
   {
-    CComBSTR pMainFileName;
-    VERIFY_SUCCEEDED(pPdbUtils->GetMainFileName(&pMainFileName));
-    VERIFY_ARE_EQUAL(pMainFileName, pMainFileName);
```

Comment:
Hah! This test was invalid before! A perk of porting to platforms with more demanding warning checks